Application/EdFi.Ods.Api/Security/Authentication/EdFiAdminAccessTokenFactory.cs

```diff
@@ -43,14 +49,22 @@ public async Task<AccessToken> CreateAccessTokenAsync(int apiClientId, string sc
 
         var @params = new
         {
-            Id = Guid.NewGuid(),
-            Expiration = DateTime.UtcNow.Add(TimeSpan.FromMinutes(_tokenDurationMinutes)),
-            Scope = scope,
-            ApiClientId = apiClientId
+            id = Guid.NewGuid(),
+            expiration = DateTime.UtcNow.Add(TimeSpan.FromMinutes(_tokenDurationMinutes)),
+            scope = scope,
+            apiclientid = apiClientId,
+            maxtokencount = _tokenPerClientLimit
```

Comment:
These parameter names are lowercase to ensure compatibility with the PostgreSQL procedure signature. This casing is also compatible with the SQL Server stored procedure because the default MSSQL server collation is case-insensitive.